demand_energy.md

```diff
@@ -23,10 +23,14 @@ gcam-version: v5.4
 | Name | Resolution | Unit | Source |
 | :--- | :--- | :--- | :--- |
 | Historical energy demand (used for calibration) | By region, technology, and year | EJ/yr | [Exogenous](inputs_demand.html) |
+| Historical floorspace demand (used for calibration) | By region and year | BM2/yr | [Exogenous](inputs_demand.html) |
 | Cost | By region, technology, and year | 1975$/GJ | [Exogenous](inputs_demand.html) |
 | Income and price elasticity | By region, demand, and year | unitless | [Exogenous](inputs_demand.html) |
+| Residential floorspace demand parameters | By region | unitless | [Exogenous](inputs_demand.html) |
+| Satiation levels | By region | m2/pers and EJ/pers | [Exogenous](inputs_demand.html) |
```

Comment:
I think "resolution" for "Satiation levels" would be better as: "By region and building type / service"

Reply:
OK!